tokio/src/runtime/builder.rs

```diff
@@ -375,6 +381,26 @@ impl Builder {
             blocking_pool,
         })
     }
+
+    /// Sets a custom timeout for a thread in the `BlockingPool`.
+    ///
+    ///
+    /// # Example
+    ///
+    /// ```
+    /// # use tokio::runtime;
+    /// # use std::time::Duration;
+    ///
+    /// # pub fn main() {
+    /// let rt = runtime::Builder::new()
+    ///     .keep_alive(Duration::from_millis(100))
+    ///     .build();
+    /// # }
+    /// ```
+    pub fn keep_alive(&mut self, duration: Duration) -> &mut Self {
+        self.keep_alive = Some(duration);
+        self
+    }
```

Comment:
This method should probably only exist when the "blocking" feature flag is enabled. Can we stick it in a
```rust
cfg_blocking_impl! {
    impl Builder {
         /// ...
         pub fn keep_alive(...) -> &mut Self { 
             // ...
         }
    }
}
```

Reply:
Using the `cfg_*` macros look relatively poor in the generated documentation due to multiple impl blocks. Perhaps this is better?
```
#[cfg(feature = "blocking")]
#[cfg_attr(docsrs, doc(cfg(feature = "blocking")))]
pub fn keep_alive(...) -> &mut Self {
```

Reply:
Oh, TIL — I thought RustDoc merged them?

Reply:
They are indeed all visible on the page, but RustDoc shows each impl block separately. On e.g. [`send_timeout`][1] I explicitly changed it to the style above to control the order in which methods appear in.

[1]: https://docs.rs/tokio/0.2/tokio/sync/mpsc/struct.Sender.html#method.send_timeout